### ISSUE TITLE
[Feature Request] Add chance

### ISSUE THREAD

**Issue by elhertz** [D1p4k/NetherPortalBoom#2]
Hey! I'm making a survival that's supposed to be really hard and frustrating and this mod caught my attention.
Could it be possible to add a chance for the explosion? I would like it to be random

**Comment by Dev0Louis** [comment]
Hmm so you mean that is just randomly explodes and randomly not?

**Comment by elhertz** [comment]
> Hmm so you mean that is just randomly explodes and randomly not?

Yes. You could also add a config so we could set a 1% to 100% chance of explosion?
